tests/run_array_analyzer_tests.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 from unittest.mock import patch
 
-import run_array_analyzer as array_analyzer
+import pysero_analyzeOD as array_analyzer
```

Comment:
Could we make it snake case please? E.g. pysero_analyze_od?

Reply:
How about `extract_od.py` or `run_od_extraction.py`? `analyzeOD` sounds to me more like interpret OD values rather than compute OD values. Also not sure if it's necessary to repeat the repo name `pysero`

Reply:
We want the CLI to be pip installable in the end like @bryantChhun figured out. So having `pysero` in the name is a good idea. How about `pysero_extract_od` for now? 

In the end both CLIs (current one and the one @smguo)  should be callable as pysero  and each phase of pipeline can become a required argument. e.g. 
`pysero --extract_od` and `pysero --analyze_binding`. 

Thoughts?

Reply:
I have added two mutually exclusive arguments: -e runs the workflow as implemented now and -a is a placeholder for analyze_od, which indeed sounds more like interpretation.

Reply:
I would suggest using subparser instead since `extract_od` and `analyze` will have different sets of arguments and should be separate:
https://stackoverflow.com/questions/17073688/how-to-use-argparse-subparsers-correctly
but this implementation works too.